Mathlib/Tactic/LibrarySearch.lean

```diff
@@ -271,7 +271,8 @@ def exact? (tk : Syntax) (required : Option (Array (TSyntax `term))) (requireClo
       for suggestion in suggestions do
         withMCtx suggestion.1 do
           addExactSuggestion tk (← instantiateMVars (mkMVar mvar)).headBeta (addSubgoalsMsg := true)
-      if suggestions.isEmpty then logError "exact? didn't find any relevant lemmas"
+      let tacticName := if requireClose then "exact" else "apply"
+      if suggestions.isEmpty then logError s!"{tacticName}? didn't find any relevant lemmas"
```

Comment:
It looks like we should just say `apply?` here, as if `requireClose` is true then an error will instead be thrown on line 269 above

Reply:
Indeed, thanks.

Reply:
A more flexible alternative is to use `{tk}` here which inserts the original syntax that the called passed in in quotes. But I don't think we need to worry about that right now

Reply:
Ah interesting, good to learn regardless thanks for the tip